### DISCUSSION TITLE
How can transfer Recent Sessions and Favorites between computers ?

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#3441] [Q&A]
Dear Sirs,
I already use 1.2.0 nightly builds for RustDesk. Recently due an error in the latest (yesterday) release all Recent Sessions were deleted...
This brings to my mind the need to transfer copy / paste Recent Sessions from other computer and if possible to transfer the Favorites  as well.
I know that the RustDesk Recent Sessions are stored at %appdata%\RustDesk\config\peers and I tried to copy a bunch of these files from another computer but it has unsuccessful.
What can I do to achieve it? Please help me.
Thanks

**Comment by mikekgr** [comment]
Dear @rustdesk can you help me on this?
Many thanks

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/discussions/2854

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/issues/2831

**Reply by mikekgr** [reply]
As I wrote in my first message, I tried that already
C:\Users\<User>\AppData\Roaming\Rustdesk\config\peers     that is essentially the same with:
%appdata%\RustDesk\config\peers
 but after copied them the RustDesk does not show them at least to my Windows 10 with the yesterday nightly build

**Reply by mikekgr** [reply]
Well the today's build is working fine in this matter thanks !

**Reply by rustdesk** [reply]
> #2831

Today's build

**Reply by rustdesk** [reply]
That's all I can help you.

**Reply by mikekgr** [reply]
THANKS you are the best supporter for RustDesk users !

**Reply by spacecake** [reply]
Sorry to bring up this old post.
As this topic is one of the first hits related to RustDesk and peers migration, I can confirm that also cross system compatibility LINUX <> Windows its a non issue.
Moving from $HOME/.config/rustdesk/peers  to %APPDATA%\RustDesk\config\peers works fine.

Cheers to the developers!

**Comment by Binly42** [comment]
cheers to the official flatpak version (yes, it was released

for guys ever using the manually installed flatpak version, seems okay to easily upgrade/migrate :
1. `flatpak install flathub com.rustdesk.RustDesk`
2. `flatpak uninstall --user com.rustdesk.RustDesk`

in case for backup, there is `~/.var/app/com.rustdesk.RustDesk/`
